src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -128,15 +128,13 @@ protected function getAttributes($object, $format = null, array $context)
             return $allowedAttributes;
         }
 
-        if (isset($context['attributes'])) {
```

Comment:
we previously skipped the cache if attributes are defined in the context. now we use the attributes as part of the cache key.